src/main/java/com/hillel/items_exchange/controller/AuthController.java

```diff
@@ -82,6 +83,13 @@
         }
     }
 
+    @PostMapping("/logout")
+    public ResponseEntity<HttpStatus> logout(HttpServletRequest req){
+        final String token = jwtTokenProvider.resolveToken(req);
+        jwtTokenProvider.invalidateToken(token);
+        return new ResponseEntity<>(HttpStatus.OK);
```

Comment:
You could make the method void and use annotation ResponseStatus(HttpStatus.OK) but it up to you.